flocker/provision/_install.py

```diff
@@ -5,13 +5,19 @@
 Install flocker on a remote node.
 """
 
-from pipes import quote as shell_quote
 import posixpath
 from textwrap import dedent
 from urlparse import urljoin
-from characteristic import attributes
+from effect import parallel, Func, Effect
```

Comment:
Despite all the lines changed, the only significant changes in this file are:
- Factoring out the code that talks ssh.
- Changing `configure_cluster` and `stop_cluster` to only return effects.